packages/cms-lib/lib/config.js

```diff
@@ -319,16 +319,15 @@ const loadConfigFromFile = (path, options = {}) => {
 const loadConfig = (
   path,
   options = {
-    ignoreEnvironmentVariableConfig: false,
+    useEnv: false,
```

Comment:
This should be `false` because we are changing it to be disabled by default and I am changing the truthiness of the variable diction.